setup.py

```diff
@@ -35,6 +37,36 @@ def __str__(self):
             language='c++'
         )
     )
+elif platform.system() == 'Darwin':
+    ext_modules.append(
+        Extension('pyvirtualcam._native_macos',
+            # Sort input source files to ensure bit-for-bit reproducible builds
+            # (https://github.com/pybind/python_example/pull/53)
+            sorted([
+                'pyvirtualcam/native_macos/main.mm',
+                'pyvirtualcam/native_macos/OBSDALMachServer.mm']),
+            include_dirs=[
+                # Path to pybind11 headers
+                get_pybind_include(),
+                'pyvirtualcam/native_macos'
+            ],
+            extra_link_args=[
+                "-framework", "AVFoundation",
+                "-framework", "AppKit",
+                "-framework", "Cocoa",
+                "-framework", "CoreFoundation",
+                "-framework", "CoreMedia",
+                "-framework", "CoreVideo",
+                "-framework", "Cocoa",
+                "-framework", "CoreMediaIO",
+                "-framework", "IOSurface",
+                "-framework", "IOKit"
+            ],
+            language='objc'
+        )
+    )
+elif platform.system() == 'Linux':
+    platform_install_requires.append("pyfakewebcam")
```

Comment:
In the [performance section of the readme](https://github.com/jremmons/pyfakewebcam#performance) it sounds like it's pretty much unusable if opencv is not installed. Maybe to avoid those issues we should just add it as dependency?

Reply:
I think I had issues with installing opencv without the distros package manager in the past (issues with linker not finding shared object or symbols).

Also, people must manually set the environment variable and `modprobe`, too.
So I think it's enough to keep the opencv performance note as part of the documentation (for now).